src/Sylius/Bundle/PaymentBundle/Resources/translations/validators.pl.yml

```diff
@@ -26,3 +26,8 @@ sylius:
       max_length: 'Nazwa metody płatności powinna mieć mniej niż {{ limit }} znaków.'
       min_length: 'Nazwa metody płatności musi składać się z co najmniej {{ limit }} znaków.'
       not_blank: Proszę podać nazwę metody płatności.
+    fee_calculator_configuration:
+      amount:
+        not_blank: Proszę wprowadzić wartość opłaty.
+      percent:
+        not_blank: Proszę wprowadzić procent opłaty.
```

Comment:
Please just add translations for `en` language, the translations are handled on [Crowdin](https://crowdin.com/project/sylius).